service/authorization/v2/authorization.proto

```diff
@@ -0,0 +1,226 @@
+// 2. one action
+// 3. one resource
+message GetDecisionRequest {
+  authorization.EntityChain entity = 1 [(buf.validate.field).required = true];
+  // name on action is required
+  policy.Action action = 2 [(buf.validate.field).required = true];
+  Resource resource = 3 [(buf.validate.field).required = true];
+
+  option (buf.validate.message).cel = {
+    id: "get_decision_request.action_name_required"
+    message: "action.name must be provided"
+    expression: "has(this.action.name)"
+  };
+}
+message GetDecisionResponse {
+  // ephemeral id for tracking between request and response
+  string ephemeral_entity_chain_id = 1;
+  // ephemeral id for tracking between request and response
+  string ephemeral_resource_id = 2;
+  // name on action is required
+  policy.Action action = 3;
+  // the decision on the resource
+  ResourceDecision decision = 4;
+  // optional list of obligations represented in URI format
+  // repeated string obligations = 5;
+}
+
+// Can the entity (PE or NPE) access?
+// 1. one entity chain
+// 2. one action
+// 3. multiple resources
+// Note: this is a more performant bulk request for multiple resource decisions
+message GetDecisionMultiResourceRequest {
+  // The entity chain must be defined
+  authorization.EntityChain entity = 1 [(buf.validate.field).required = true];
+  // name on action is required
+  policy.Action action = 2 [(buf.validate.field).required = true];
+  repeated Resource resources = 3 [
+    (buf.validate.field).required = true,
+    (buf.validate.field).repeated = {min_items: 1}
+  ];
+
+  option (buf.validate.message).cel = {
+    id: "get_decision_multi_request.action_name_required"
+    message: "action.name must be provided"
+    expression: "has(this.action.name)"
+  };
+}
+message GetDecisionMultiResourceResponse {
+  // ephemeral id for tracking between request and response
+  string ephemeral_entity_chain_id = 1;
+  // name on action is to be relied upon
+  policy.Action action = 2;
+  // convenience flag indicating global resource decisions result (permit/deny)
+  google.protobuf.BoolValue all_permitted = 3;
+  // individual resource decisions
+  repeated ResourceDecision resource_decisions = 4;
+}
+
+// Can the entities (PE or NPE) access?
+// 1. Multiplexing of a Decision<entityChain, Action, repeated Resource> request
+// This is a more performant bulk request for complex decisioning (multiple entity chains or actions on
+// multiple resource decisions)
+message GetDecisionBulkRequest {
+  repeated GetDecisionMultiResourceRequest decision_requests = 1;
+}
+message GetDecisionBulkResponse {
+  repeated GetDecisionMultiResourceResponse decision_responses = 1;
+}
+
+// What can the entity do, broken down as permitted actions on attribute value FQNs
+//
+// Note: the v1 API parameter 'scope' has been dropped, and it is recommended to use
+// GetDecision if the resource is known
+message GetEntitlementsRequest {
+  // list of requested entities
+  // todo: should this be an entity chain?
```

Comment:
Will not deciding this now cause a need for a v3 service?

Reply:
I'm still not sure if this should be a single Entity or a chain, but the idea of a chain is that it's something ERS builds

Reply:
allow a chain or a token we resolve into a chain within auth service